src/bot/lib/modifiers/register-commands.js

```diff
@@ -43,22 +43,14 @@ export function registerCommands(...commandsAndAliases) {
       })
 
       if (typeof result === 'string') {
-        return {
-          ...messageContext,
-          message: result,
-        }
+        return { message: result }
       }
 
       return result
     } catch (err) {
       console.error(`Something went wrong processing command ${commandName}`, err)
 
-      return {
-        ...messageContext,
-        // This typo is intentional. This is the sanpe-bot.
-        // "sanpe" looks like "santé" which is the french word for health
-        message: 'The commander should check my healph',
-      }
+      return { message: 'The commander should check my healph' }
```

Comment:
I think it should set fulfilled at true at this point too. Since a handler has take the job and failed

Reply:
`fulfilled` is by default equal to `true`. See https://github.com/mindsers/sanpe-bot/pull/17/files#diff-f4fd88ee249fe934c063de2e756fd76d0a610ab3ad0068edb1dbbe870344b387R26